src/test/regress/expected/create_drop_database_propagation.out

```diff
@@ -661,7 +657,7 @@ SELECT result from run_command_on_all_nodes(
 ) ORDER BY result;
  result
 ---------------------------------------------------------------------
- t
+ f
```

Comment:
all the changes in this file happens bc now we don't propagate grant for non-distributed databases, and while CREATE is granted by default, the other three permissions are not. That's why while we return false for CREATE, we return true for the others.